draft-bbs-signatures.md

```diff
@@ -802,6 +802,22 @@ This section defines the format for a BBS ciphersuite. It also gives concrete ci
 
 ## Ciphersuite Format
 
+### Ciphersuite ID
+
+The following section defines the format of the unique identifier for the ciphersuite denoted `Ciphersuite_ID`. The REQUIRED format for this string is
+
+```
+  "BBS_" || H2C_SUITE_ID || ADD_INFO
```

Comment:
```suggestion
  BBS || "_" || H2C_SUITE_ID || "_" || ADD_INFO || "_"
```
It may be better to define the ID to end with a `_` so we will not have to define that ADD_INFO MUST end with an `_`. Also there is an `_` that is needed after the h2c id i think.

Reply:
I think most of the H2C_SUITE_ID's follow with an `_` already? hence I was just following the pattern established by the BLS spec?

Reply:
See https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-hash-to-curve-14#section-8.8

Reply:
quite right! my apologies. So the _ after the H2C_SUITE_ID is not needed. I updated the suggestion.

Reply:
What about if ADD_INFO is not supplied though? Then it would be a double underscore would it not?

Reply:
Yea, which could provide a way for a verifier to know when ADD_INFO was not supplied?? I do realize that this is unconventional so Im fine with the current approach also.